benchmark/README.md

```diff
@@ -0,0 +1,28 @@
+# Aerospike PHP client benchmark
+
+This guide provides step-by-step instructions on running the benchmark for the Aerospike php client.
+
```

Comment:
Please add a brief description of what the benchmark does here - what operations are used, concurrency, transaction volume, and what namespace/set it will write into.